.github/workflows/deploy-test-smartContract.yml

```diff
@@ -0,0 +1,45 @@
+name: Smart contract example Deploy
+
+on:
+  [push]
+
+
+jobs:
+  deploy-SC-example:
+    defaults:
+      run:
+        working-directory: ./examples/smartContracts/
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install node
+        uses: actions/setup-node@v3
+        with:
+          node-version: "18"
+          cache: "npm"
+          cache-dependency-path: ./examples/smartContracts/contracts/package-lock.json
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Build
+        run: npm run build
+
+      - name: Execution of tests
+        run: |
+          echo ${{ secrets.JSON_RPC_URL_PUBLIC }}
+          if npm run test-smart-contract-example ; then
+            echo "Contracts successfully deployed!"
+          else
+            echo "Failed to deploy contracts ..."
+            exit 1
+          fi
+          env:
+            JSON_RPC_URL_PUBLIC: ${{ secrets.JSON_RPC_URL_PUBLIC }}
+            JSON_RPC_URL_PRIVATE: ${{ secrets.JSON_RPC_URL_PRIVATE }}
+            DEPLOYER_SECRET_KEY: ${{ secrets.DEPLOYER_SECRET_KEY }}
+            RECEIVER_SECRET_KEY: ${{ secrets.RECEIVER_SECRET_KEY }}
```

Comment:
Why do you need a key on the receiver side ?

Reply:
It is indeed almost unnecessary, but we still need it because we are adding the receiverAccount to the wallet to test the function `addSecretKeysToWallet`

Reply:
Why don't you test that with DEPLOYER_SECRET_KEY ? If not possible, I will create another wallet

Reply:
deployerAccount is already the baseAccount so we can't test with it. I've renamed the `RECEIVER_SECRET_KEY` to `OTHER_ACC_SECRET_KEY`,  which will also be used as a receiver.